### DISCUSSION TITLE
How to specify custom `stringify` and `parse` functions for JSON table (entity) properties?

### DISCUSSION THREAD

**Discussion by supfiger** [mikro-orm/mikro-orm#5450] [Q&A] [answered]
I tried to create a new class (let's call it `AppJsonType`), extending from `JsonType` class, overriding `convertToJSValue` and `toJSON` methods, and then use it inside properties, like `@Property({ type: AppJsonType })`.

And also I tried to provide a custom Driver with a custom Platform, overriding  `convertJsonToDatabaseValue`, 
`convertsJsonAutomatically`, and `convertJsonToJSValue` methods.

And also I tried both. 

But looks like it does not work correctly. Probably there is auto JSON parsing somewhere that I didn't override.

Is there a way to do it?

### Drivers: 
Postgres and Better SQLite.

**Comment by B4nan** [comment]
Provide complete repro so its clear what you are doing (and mainly what you are up to, as this sounds like X/Y problem). 

Note that namely postgres does convert JSON automatically - on the driver level, that happens outside of the ORM.

**Reply by supfiger** [reply, marked as answer]
> postgres does convert JSON automatically

Thanks about mentioning it.

So I solved the issue by extending not from `JsonType` class, but from `TextType` class.
